### PR TITLE
Spellchecking: typo fixes and initial spell.ignore

### PR DESCRIPTION
**Pull request by ana**
I'm fixing a bunch of typo and adding an initial spell.ignore file.
This PR should help https://github.com/autotest/tp-libvirt/pull/3735 to be merged

### CODE REVIEW

**Comment by chunfuwen** [comment]
@ana 
Great job you done, thank you for huge effort you put on to correct spelling.

**Comment by chunfuwen** [comment]
libvirt/tests/cfg/remote_access/remote_with_ssh.cfg
    libvirt/tests/cfg/remote_access/remote_with_tcp.cfg 
   libvirt/tests/cfg/remote_access/remote_with_tls.cfg

These three cfg files has changes in variable value, we may need double check whether we need update python code or CI config accordingly

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by Yingshun — APPROVED** [review]

**Review by chloerh — APPROVED** [review]
Thank you so much for helping us with spell check!

File: libguestfs/tests/virt_win_reg.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def run(test, params, env):
         if operation == "query":
             output_in_guest = output.split()[-1].strip()
             if not output_in_guest == output_virt_win_reg:
-                test.fail("Informations are not equal from "
+                test.fail("Information are not equal from "
```
Comment (kylazhang) [inline libguestfs/tests/virt_win_reg.py:132]: Should be "Information is xxx"?

File: libvirt/tests/src/migration/migrate_vm.py [hunk 2]
```diff
@@ -911,7 +911,7 @@ def check_migration_disk_port(params):
     """
     Handle the option '--disks-port'.
     As the migration thread begins to execute, this function is executed
-    at same time almostly. It will wait for several seconds to make sure
+    at same time almost. It will wait for several seconds to make sure
```
Comment (kylazhang) [inline libvirt/tests/src/migration/migrate_vm.py:914]: "almost at the same time" ?

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py [hunk 3]
```diff
@@ -410,7 +410,7 @@ def check_migration_timeout_suspend(timeout):
         """
         Handle option '--timeout --timeout-suspend'.
         As the migration thread begins to execute, this function is executed
-        at same time almostly. It will sleep the specified seconds and check
+        at same time almost. It will sleep the specified seconds and check
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py:413]: "almost at the same time" ?

File: libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_disk.py [hunk 4]
```diff
@@ -414,7 +414,7 @@ def run(test, params, env):
 
         # Test delete snapshot without "--metadata", delete external disk
         # snapshot will fail for now.
-        # Only do this when snapshot creat succeed which filtered in cfg file.
+        # Only do this when snapshot create succeed which filtered in cfg file.
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_disk.py:417]: "snapshot is created successfully"?
"Which is filtered"

File: libguestfs/tests/virt_win_reg.py (hunk 1 above)
Comment (ana) [reply libguestfs/tests/virt_win_reg.py:132]: @kylazhang feel free to send a PR after fixing the grammar! This was about typo only.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by kamilvarga — COMMENTED** [review]
Others LGTM.

File: libvirt/tests/src/multiqueue.py [hunk 5]
```diff
@@ -105,7 +105,7 @@ def top_vhost(test, pids, expected_running_vhosts=1, timeout=15):
             if timeout > 0:
                 timeout -= 3
                 time.sleep(3)
-                logging.debug("Trying again to avoid occassional...")
+                logging.debug("Trying again to avoid occasional...")
```
[the remark this answers is not in the document]
Comment (ana) [reply libvirt/tests/src/multiqueue.py:108]: @kamilvarga  feel free to send a PR after fixing the grammar! This was only about typos.

**Review by kamilvarga — APPROVED** [review]

**Review by kamilvarga — APPROVED** [review]
LGTM.

File: libguestfs/tests/virt_win_reg.py (hunk 1 above)
Comment (kylazhang) [reply libguestfs/tests/virt_win_reg.py:132]: ok

**Review by kylazhang — APPROVED** [review]

**Pull request merged**